File: python/mqttClient.py
import time
import paho.mqtt.client as mqtt

def on_subscribe(mosq, obj, mid, granted_qos):
    print("Subscribed: " + str(mid))

def on_message(mosq, obj, msg):
    print(msg.topic + " " + str(msg.qos) + " " + str(msg.payload))

def on_connect(mosq, obj, rc):
    mqttc.subscribe("/beacons", 0)
    print("Connected")

mqttc = mqtt.Client()
mqttc.on_connect = on_connect
mqttc.on_subscribe = on_subscribe
mqttc.on_message = on_message
mqttc.connect("mqtt.bconimg.com", 1883, 60)
mqttc.loop_forever()

File: php/mqtt-client.php
<?php


$mqttclient=new Mosquitto\Client;

$mqttclient->onConnect(function() use ($mqttclient){
	echo "connect success \n";
	$mqttclient->subscribe('/beacons',0);
});

$mqttclient->onMessage(function($message)use($mqttclient){
	var_dump($message);
	echo "=================================\n";
});

$mqttclient->onLog(function($level,$msg)use($mqttclient){
//	echo "\n";
//	echo $msg;
//	echo "\n";
});

$mqttclient->connect('mqtt.bconimg.com',1883,60);

$mqttclient->loopforever();
